src/utilities/time.test.ts

```diff
@@ -40,6 +41,20 @@ test('convertDurationStringToUs', () => {
     `);
 });
 
+test('convertUTCtoMs', () => {
+  // standard date conversion
+  expect(convertUTCtoMs('2024-01-01T00:00:00Z')).toEqual(1704067200000);
```

Comment:
Do you mind adding a case for when a time zone isn't supplied for the standard date format too?